dev-client/datatypes/sites.ts

```diff
@@ -0,0 +1,17 @@
+export type Point2D = [number, number];
+
+export class DisplaySite {
+    lat: number;
+    lon: number;
```

Comment:
question: can we re-use a `coordinate` type from https://www.npmjs.com/package/@types/geojson? `rnmapbox` also depends on this library and expects this format so it might make interop code a bit nicer

Reply:
Yes, agreed. I ended up doing this in my next PR.